build/cloud/README.md

```diff
@@ -1,36 +1,11 @@
 ##  Deploy the cloud part into a k8s cluster
+Please refer to the README.md in the directory v1.3 or pre1.3.
 
-This method will guide you to deploy the cloud part into a k8s cluster,
-so you need to login to the k8s master node (or where else if you can
-operate the cluster with `kubectl`).
+If you want to know more, please refer to [CHANGELOG-1.3.md](https://github.com/kubeedge/kubeedge/blob/master/CHANGELOG-1.3.md#kubeedge-v13-release-notes)
 
-The manifests and scripts in `github.com/kubeedge/kubeedge/build/cloud`
-will be used, so place these files to somewhere you can kubectl with.
+* For KubeEdge v1.3, it supports **CloudCore HA** and **EdgeNode auto TLS Bootstrapping** and so on:
+  * Single CloudCore: Refer to [README.md](v1.3/README.md)
```

Comment:
Shall we keep the main mode files under build/cloud directly?
Please note that https://github.com/kubeedge/kubeedge/pull/1781 is relying the yaml files

Reply:
@kevin-wangzefeng  ok, I will modify base on #1781